docs/CreateAKSClusterForScalarDB.md

```diff
@@ -21,32 +21,38 @@ When deploying ScalarDB Server, you must:
 
 The following are some recommendations for deploying ScalarDB Server. These recommendations are not required, so you can choose whether or not to apply these recommendations based on your needs.
 
+### Create at least three worker nodes and three pods
```

Comment:
It's a comment on the whole doc.
I'm wondering if this should be a deprecate document since the corresponding on for EKS is marked as `Deprecated`.
Can you clarify? (maybe it's related to the reason why we don't have updates for ScalarDB Cluster doc for AKS?)

Reply:
@feeblefakie  
Thank you for your question!

This question is related to the ScalarDB Cluster's listing in each marketplace.
At this time, we provide ScalarDB Cluster on only AWS Marketplace as follows.

* AWS Marketplace
  * ScalarDB Cluster (Pay As You Go)
  * [Deprecated] ScalarDB Server (BYOL)
* Azure Marketplace
  * ScalarDB Server (BYOL)

So, if we mark ScalarDB Server as deprecated on the Azure side, basically, Azure users cannot use ScalarDB in any way. This is why I don't still mark ScalarDB Server as deprecated on the Azure Marketplace (Azure document) side.

If it (Azure users cannot use ScalarDB) is acceptable, we can mark it as deprecated.
(Of course, from the technical perspective, users can use ScalarDB Server on Azure environment even if we mark it as a deprecated product.)

Also, there are almost no users on the Azure Marketplace (ScalarDB Server BYOL), so it might be no problem if we mark it as deprecated.
What do you think?

---

For your information, I will organize the Marketplace and documents as follows after we will release license based BYOL.

* AWS Marketplace
  * ScalarDB Cluster (Pay As You Go)
  * ScalarDB Cluster (license-key based BYOL)
  * [Deprecated] ScalarDB Server (BYOL)
* Azure Marketplace
  * ScalarDB Cluster (license-key based BYOL)
  * [Deprecated] ScalarDB Server (BYOL)

In this case, both users of AWS and Azure can use ScalarDB Cluster on each cloud environment.

Reply:
@feeblefakie  
As we discussed, now we are preparing to provide ScalarDB Cluster on Azure Marketplace.
So, I will mark `ScalarDB Server` as `Deprecated` in all documents in another PR.

Reply:
@kota2and3kan Got it. Thank you!